build.gradle

```diff
@@ -30,6 +33,7 @@ allprojects {
         google()
         jcenter()
         maven { url "https://dl.bintray.com/wordpress-mobile/maven" }
+        maven { url "https://jitpack.io" }
```

Comment:
Could you please double check if we really need this. Thanks!

Reply:
It wasn't needed. Removed 3b2749e.